.github/workflows/test.yml

```diff
@@ -25,7 +25,10 @@ jobs:
         args: --verbose
 
   test-unit:
-    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        os: [ ubuntu-20,04, macos-14 ]
```

Comment:
may be adding the support of freebsd, this can be in a different PR :)

Reply:
Out of scope of current PR, yep.

Reply:
But we still need to ensure that clown work as intended on freebsd

Reply:
I created https://github.com/fahedouch/go-logrotate/pull/52 that adds FreeBSD CI.